### PR TITLE
Disable Widgets for ARC devices

### LINKED ISSUE #7525: Disable widget pop up for Chrome OS devices
Chromebooks (running ARC) can't show widgets so we don't want to show a widget promo for them.

### PR DESCRIPTION
**Pull request by kwonye**
Fixes #7525 

To test:

1. Install on Chrome OS device
2. Verify that the widget install promo dialog doesn't show up after visiting stats three times.
3. Verify that the widget install promo DOES show up on an android device

I have verified that this check is the correct one, even though it's unofficial.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java [hunk 1]
```diff
@@ -621,8 +622,8 @@ private void bumpPromoAnalyticsAndShowPromoDialogIfNecessary() {
         AppPrefs.bumpAnalyticsForStatsWidgetPromo();
 
         // Should we display the widget promo?
-        int counter = AppPrefs.getAnalyticsForStatsWidgetPromo();
-        if (counter == 3 || counter == 1000 || counter == 10000) {
+
+        if (shouldShouldWidgetPromo()) {
```
Comment (roundhill) [inline WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java]: I think you meant `shouldShowWidgetPromo()`?

**Comment by roundhill** [comment]
Also, not sure what's up w/ Travis? 🤷‍♂️

File: WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java (hunk 1 above)
Comment (kwonye) [reply WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java]: ... yes, thanks for that.

**Comment by kwonye** [comment]
It was a codestyle issue, my fault. Thanks for the review!

**Pull request merged**